torch/_dynamo/eval_frame.py

```diff
@@ -360,9 +359,79 @@ def first_real_inst_idx(code):
     raise RuntimeError("RESUME instruction not found in code")
 
 
+# Converts a CodePart structure, which carries our
+# re-compilation code, source, and more, into a
+# plan for how to mark certain dims as dynamic.
+def dynamic_plan_from_code_part(code_part, dynamic_plan):
+    from torch._dynamo.source import TensorProperty, TensorPropertySource
+
+    if code_part is None or not config.automatic_dynamic_shapes:
+        return
+
+    def is_dynamic_source_candidate(source):
+        if isinstance(source, TensorPropertySource):
+            prop = source.prop
+            # TODO: Expand to stride?
```

Comment:
Yes, you need to do every TensorProperty we can generate a symbol for.

Reply:
Sg